step.yml

```diff
@@ -298,10 +300,12 @@ inputs:
   - should_retry_test_on_fail: "no"
     opts:
       category: Debug
-      title: "Should retry test on failure?"
+      title: "Should retry tests on failure? (Not available in Xcode 13+)"
+      summary: If you set this input to `yes`, the Step will rerun your tests in case of failure.
       description: |-
-       If you set this input to `yes`, the Step will rerun your tests. With Xcode 13 and above, only your failed test cases will be rerun. With older Xcode versions, all test cases will be rerun.
-        Please note that this feature is only available from Xcode 13 and above, earlier versions support rerunning ALL test cases instead of a specific failed one.
+        If you set this input to `yes`, the Step will rerun your tests once in case of failure. Note that this will rerun ALL of your tests and just the ones that failed.
```

Comment:
how about this: "If you set this input to 'yes', the Step will rerun ALL your tests once in the case of failed test/s. Note that ALL your tests will be rerun, not just the ones that failed."